### PR TITLE
Fixed several warnings.

### PR DESCRIPTION
**Pull request by mpimenov**

### CODE REVIEW

File: map/map_tests/ge0_parser_tests.cpp [hunk 1]
```diff
@@ -223,7 +221,7 @@ UNIT_TEST(LatLonFullAndClippedCoordinates)
     }
   }
 
-  for (int coordSize = 1; coordSize <= 8; ++coordSize)
+  for (size_t coordSize = 1; coordSize <= 8; ++coordSize)
```
Comment (VladiMihaylenko) [inline map/map_tests/ge0_parser_tests.cpp]: uint8_t ?

File: map/map_tests/ge0_parser_tests.cpp [hunk 2]
```diff
@@ -81,13 +80,12 @@ bool ConvergenceTest(double lat, double lon, double latEps, double lonEps)
...
 }
-
-}  // unnamed namespace
+}  // namespace
 
 UNIT_TEST(Base64DecodingWorksForAValidChar)
 {
   Ge0ParserForTest parser;
-  for (size_t i = 0; i < 64; ++i)
+  for (int i = 0; i < 64; ++i)
```
Comment (VladiMihaylenko) [inline map/map_tests/ge0_parser_tests.cpp]: uint8_t?

File: map/map_tests/ge0_parser_tests.cpp (hunk 1 above)
Comment (mpimenov) [reply map/map_tests/ge0_parser_tests.cpp]: No, why?
uint8_t is still too big, so I prefer to stick to a more common type.

File: map/map_tests/ge0_parser_tests.cpp (hunk 2 above)
Comment (mpimenov) [reply map/map_tests/ge0_parser_tests.cpp]: That would require changing the signature of the obscure MapsWithMe_Base64Char function. I'd rather not.

Comment (VladiMihaylenko) [reply map/map_tests/ge0_parser_tests.cpp]: ok

**Pull request merged**